docs/image-verification.md

```diff
@@ -0,0 +1,23 @@
+# Image Verification
+
+To enable image verification, add a stanza like the following to the conainerd config:
+
+```yaml
+[plugins]
+  [plugins."io.containerd.image-verifier.v1.bindir"]
+    enabled = true
+    bin_dir = "/opt/containerd-image-verifier/bin"
+```
+
+Place one or more verifier binaries in `bin_dir` which conform to the following API:
```

Comment:
From a CRI perspective: binary plugins like this will have their resources accounted in (and constrained by) containerd's cgroups rather than a pod's.  I don't think we need to solve this in the first iteration, but we may want to think about possibilities for charging resource usage to a pod.